framework/Source/Kirakira.swift

```diff
@@ -137,3 +217,114 @@ private extension Kirakira {
         sparklesEffect.centerSaturation = centerSaturation * Float(colorMode.rawValue)
     }
 }
+
+public extension Kirakira.Parameters {
+
+    enum DecodingError: Error {
+        case jsonDataEncodingFailed
+    }
+
+    init(with jsonString: String) throws {
+        guard let data = jsonString.data(using: .utf8) else {
+            throw DecodingError.jsonDataEncodingFailed
+        }
+        try self.init(with: data)
+    }
+
+    init(with jsonData: Data) throws {
+        let decoder = JSONDecoder()
+        let parameters = try decoder.decode(Kirakira.Parameters.self, from: jsonData)
+        self = parameters
+    }
+}
```

Comment:
Wonder where `init(with jsonString: String)` is used?

I think we probably don't need these two convenience methods as the caller should've known the data is in JSON format and explicitly called `JSONDecoder().decode(Kirakira.Parameters.self, from: jsonData)`.

Reply:
Hi @peteranny 
It came from the discussion between YY and me about the constructor of the Kirakira.Parameters, but it wasn't been used at last.

Hi @yyjim, 
I'm going to remove `init(with jsonString: String)`. 
Please let me know if you have any other concerns; thanks!